lib/config/types.ts

```diff
@@ -442,6 +442,8 @@ export interface RenovateOptionBase {
    * This is used to add depreciation message in the docs
    */
   deprecationMsg?: string;
+
+  regexOrGlob?: boolean;
```

Comment:
```suggestion
  patternMatch?: boolean;
```
This is more generic, when we evolve this in the future.

Reply:
Not sure about this, aren't regex patterns too.

Reply:
The docs page is called String Pattern Matching and contains regex and glob, so I think this should be clear. 
https://docs.renovatebot.com/string-pattern-matching/